tests/databases/api_tests.py

```diff
@@ -801,3 +803,26 @@ def test_get_database_related_objects_not_found(self):
         uri = f"api/v1/database/{database.id}/related_objects/"
         rv = self.client.get(uri)
         self.assertEqual(rv.status_code, 404)
+
+    def test_export_database(self):
+        """
+        Database API: Test export database
+        """
+        self.login(username="admin")
+        uri = "api/v1/database/1/export/"
+        rv = self.client.get(uri)
+
+        self.assertEqual(rv.status_code, 200)
+
+        buf = BytesIO(rv.data)
+        self.assertTrue(is_zipfile(buf))
+
+    def test_export_database_not_allowed(self):
+        """
+        Database API: Test export database
+        """
+        self.login(username="gamma")
+        uri = "api/v1/database/1/export/"
+        rv = self.client.get(uri)
+
+        self.assertEqual(rv.status_code, 401)
```

Comment:
This should return a 404, right?

Reply:
nit: add one more test with a non existent database_id (use max then plus 1).

Reply:
Hmmm, I think 403 would be more appropriate, no? The user is authenticated (as "gamma") but not authorized.

I'll add another test for an non-existent database returning 404.

Reply:
Oh! I was misunderstanding this!
It's actually denied because of FAB's permissions. I was thinking that it was a test to infer the user had no access to view/list/get that particular database, btw that's a good test to add also: https://github.com/apache/incubator-superset/blob/master/tests/databases/api_tests.py#L542